qr-code/src/main/java/com/adyen/checkout/qrcode/QRCodeComponent.kt

```diff
@@ -84,36 +71,24 @@ class QRCodeComponent(
             redirectDelegate.makeRedirect(activity, action.url)
```

Comment:
I am rethinking this part here, the logic is that the QR code component might sometime launch a redirect instead of the status polling. In this instance it's the component that decides which action to make so it can be done here.
However when we go to the 3DS2 component a redirect can be done mid flow and surely the 3DS2 delegate is the one that decides if and when a redirect should happen. Maybe we'll figure this out when we work on the 3DS2 delegate but just leaving the comment here in case we want to come back later and change the logic here as well.